changelog.md

```diff
@@ -314,6 +314,9 @@
 
 - `--hint:CC` now goes to stderr (like all other hints) instead of stdout.
 
+- Tuple expressions are now parsed consistently as
```

Comment:
how about move this to `Language changes` ? in future work we should clarify in the template `changelogs/changelog_X_XX_X.md` what belongs in each category (with examples)

Reply:
Yeah, I thought about moving it to `Language changes` too; if you agree then I'll do it.